ppp_datamodel/communication/request.py

```diff
@@ -24,6 +24,10 @@ def _check_attributes(self, attributes, extra=None):
             raise AttributeNotProvided('Missing attributes: %s' % ', '.join(missing_attributes))
         if not isinstance(attributes['language'], basestring):
             raise TypeError('"language" attribute is not a string.')
+        if 'response_language' not in attributes:
+            attributes['response_language'] = attributes['language']
```

Comment:
Python tip: These two lines are equivalent to `attributes.setdefault('response_language', attributes['language'])